src/TaskRunner.php

```diff
@@ -144,11 +144,38 @@ private function createConfiguration()
             __DIR__.'/../config/runner.yml',
             'runner.yml.dist',
             'runner.yml',
+            $this->getLocalConfigurationFilepath(),
         ], $config);
 
         return $config;
     }
 
+  /**
+   * Get the local configuration filepath.
+   *
+   * @param string $configuration_file
+   *   The default filepath.
+   *
+   * @return string|null
+   *   The local configuration file path, or null if it doesn't exist.
+   */
+    private function getLocalConfigurationFilepath($configuration_file = 'openeuropa/taskrunner/runner.yml')
+    {
+        if ($config = getenv('TASKRUNNER_CONFIG')) {
```

Comment:
I love that this option is provided, it gives ultimate flexibility regarding the location of the config file. However I am wondering about the naming, what if there are many different task runners available on the system? Wouldn't a better name for the variable be `OPENEUROPA_TASKRUNNER_CONFIG`?

Reply:
Why not!

Reply:
Fixed.